dll/ime/msctfime/lang/tr-TR.rc

```diff
@@ -0,0 +1,23 @@
+/*
+ * PROJECT:     ReactOS msctfime.ime
+ * LICENSE:     LGPL-2.1-or-later (https://spdx.org/licenses/LGPL-2.1-or-later)
+ * PURPOSE:     Turkish resource file
+ * TRANSLATOR:  Copyright 2024 Ethem De Santa (scropool) <oyunyumru4@gmail.com>
+ */
+
+LANGUAGE LANG_TURKISH, SUBLANG_DEFAULT
+
+STRINGTABLE
+BEGIN
+    IDS_OK               "&Tamam"
+    IDS_CANCEL           "&İptal"
+    IDS_ABORT            "&Durdur"
+    IDS_RETRY            "&Yeniden dene"
+    IDS_IGNORE           "Y&oksay"
+    IDS_YES              "&Evet"
+    IDS_NO               "&Hayır"
+
+    IDS_ENTER            "&Gir"
+    IDS_FINALIZE_STRING  "D&izeyi sonlandır"
```

Comment:
Note: Most of the time MS Windows tries to avoid &i because it is hardly readable. Are you sure that MS uses &i here (you can prove that with a pic). If not, then chosing a different letter than &i might be preferable.

Reply:
Resolved, and superseded by stasms comment.

Reply:
@scropool Try changing it with "Di&zeyi sonlandır"